documentation/source/user_section/installation/mac.rst

```diff
@@ -203,16 +183,19 @@ First, save your Docker image:
 
    .. code:: bash
 
-      docker commit <CONTAINER_ID> <YOUR_NAME>/<DISTROS>:<VERSION>
+      docker commit <CONTAINER_ID> <IMAGE_NAME>/ubuntu:ubuntu22.04
 
 Create an X11 server for handling display:
 
 1. Install XQuartz X11 server.
-2. Check ‘Allow connections from network clientsoption inXQuartz\` settings.
+2. Check ‘Allow connections from network clients option in XQuartz\` settings.
 3. Quit and restart XQuartz.
 4. In XQuartz window xhost + 127.0.0.1
 5. In your other Terminal window, run:
-   ``docker run -e DISPLAY=host.docker.internal:0 -it <CONTAINER_ID>``
+   
+   .. code:: bash 
+
+      docker run -e DISPLAY=host.docker.internal:0 -it <IMAGE_NAME>/ubuntu:ubuntu22.04
 
```

Comment:
Adding a note here to add the same `sct_check_dependencies` step to the other pages:

```suggestion

6. You can test whether GUI scripts are available by running the following command in your Docker container:
 
   .. code:: bash
   
      sct_check_dependencies
      
   You should see two green ``[OK]`` symbols at the bottom of the report for "PyQT" and "matplotlib" checks, which represent the GUI features provided by SCT. 
   
   You may see some warnings about "current screen resources", "XDG_RUNTIME_DIR", or "FBConfigs". These are non-critical warnings, and should not affect the performance of the GUI scripts.
```